airflow/www/templates/airflow/confirm.html

```diff
@@ -28,11 +28,17 @@ <h2>Wait a minute.</h2>
       <pre><code>{{ details }}</code></pre>
     {% endif %}
   </div>
-  <form method="POST">
+  {% if endpoint %}
+    <form method="POST" action="/{{ endpoint }}">
```

Comment:
This won't work if Airflow is not rooted at `/` -- endpoint should converted to a URL by the view function using `url_for` and the template can then just print it.